contributors/guide/README.md

```diff
@@ -73,12 +73,16 @@ You get the idea - if you ever see something you think should be fixed, you shou
 
 There are multiple repositories within the Kubernetes community and a full list of repositories can be found [here](https://github.com/kubernetes/).
 Each repository in the Kubernetes organization has beginner-friendly issues that provide a good first issue. For example, [kubernetes/kubernetes](https://git.k8s.io/kubernetes) has [help wanted](https://go.k8s.io/help-wanted) and [good first issue](https://github.com/kubernetes/kubernetes/labels/good%20first%20issue) labels for issues that should not need deep knowledge of the system.
-The `good first issue` label indicates that members have committed to providing extra assistance for new contributors.
+The `good first issue` label indicates that members have committed to providing extra assistance for new contributors. Read more [here](/contributors/devel/help-wanted.md).
 <!-- TODO: review removing this note after 3 months or after the 1.12 release -->
 Please note that while several of the repositories in the Kubernetes community have `good first issue` labels already, they are still being applied throughout the community.
 
 Another good strategy is to find a documentation improvement, such as a missing/broken link, which will give you exposure to the code submission/review process without the added complication of technical depth. Please see [Contributing](#contributing) below for the workflow.
 
+#### Issue Assignment in Github
+
+Often, new contributors ask to be assigned an issue they are willing to take on. Unfortunately, due to GitHub limitations we can only assign issues to [org members](#community) or repo collaborators. Instead, a code owner will remove any `help-wanted` or `good-first-issue` flags and verbally state in an issue comment that it is yours to work on.
```

Comment:
I think that this:
> a code owner will remove any `help-wanted` or `good-first-issue` flags and verbally state in an issue comment that it is yours to work on.

may need more communicating to code owners, so they actually do this. 

I'd also be slightly hesitant to remove them as often new contributors say "hey i'll work on this" and then the issue is forgotten without anything happening (perhaps they don't actually wind up having time?) and then the issue is still unresolved and no longer findable as a `good-first-issue` or `help-wanted`. OTOH if the labels are still there the "I'll work on this comment" hopefully signals to other contributors that someone plans to work on it (and the comments have visible timestamp to help somewhat with identifying staleness) ...? 🤔

Reply:
^I was thinking this, but I didn't write it down. Trusting people to be a good sport about work. If someone else sees a comment with no follow-up, there should be a rather short time-out (~1 week) before someone else can take the issue away.

Reply:
Yeah, I was on the fence abou this myself but I've seen the reviewer?project owner do this often enough that I thought it was good practice. We can leave it off for now and leave it up to the maintainer/reviewer.